docs/plugins/albumtypes.rst

```diff
@@ -0,0 +1,57 @@
+AlbumTypes Plugin
+=================
+
+The ``albumtypes`` plugin adds the ability to format and output album types,
+such as "Album", "EP", "Single", etc. List of available type can be found
+`here`_.
```

Comment:
Maybe for this last link, I'd word it as: "For the list of available album types, see the MusicBrainz documentation." (with the last two words linked)